integration_tests/outdated-pkgs/pkgr-update.yml

```diff
@@ -0,0 +1,16 @@
+Version: 1
+# top level packages
+Packages:
+  - R6
+  - pillar
+  - Matrix
```

Comment:
this is a heavy package IIRC and takes a bit of compile time to install, probably better to choose something simpler. What was the reason to add Matrix into the mix?

Reply:
Yeah we can probably remove Matrix, see my comment here: https://github.com/metrumresearchgroup/pkgr/pull/367#issuecomment-884409620